### ISSUE TITLE
Any way to disable screen reader only messages?

### ISSUE THREAD

**Issue by kunokdev** [airbnb/react-dates#893]
Is threre any way to disable screen reader only messages? 

![image](https://user-images.githubusercontent.com/13220754/33783167-e108302e-dc5b-11e7-9ea5-4271a45cceb5.png)![image](https://user-images.githubusercontent.com/13220754/33783167-e108302e-dc5b-11e7-9ea5-4271a45cceb5.png)

The problem with these messages is that they ruin SEO heavily. Basically my homepage is very minimal and there is only a small amount of text other than date picker. 
Now these screen reader messages print text (I am server-side rendering these) and these words become most used keywords. I get SEO warnings all over the place because of them.

Is there any way to disable these messages or does anybody know a proper workaround?

Thanks!

**Comment by majapw** [comment]
Um, it seems bad to sacrifice accessibility for SEO but @backwardok perhaps has some insight as to how the two can exist in tandem. 

As a hack, although I don't recommend it, each of these phrases reads from a `phrases` object that you can override via the `phrases` prop. I imagine if you were to pass an empty string for the text you didn't want to render, that would work.

**Comment by kunokdev** [comment]
@majapw In my case SEO has higher priority since my website is not optimized for accessibility overall at the moment, while I have huge issues with SEO optimization due to lack of rich content. I apologize in front for lack of respect towards accessibility.

The hack worked great, I just passed empty object to `phrases` prop and it worked like a charm. Thank you so much for this awesome and flexible library, you guys are doing amazing work!

I am looking forward if @backwardok has any suggestion on how to make them coexist.

**Comment by backwardok** [comment]
Ah sorry for not responding earlier!

In general, I would say that we shouldn't sacrifice accessibility for SEO. However, I do find the content that gets read out, and the general instructions, to be somewhat problematic for accessibility in that the instructions are only valid for a keyboard user who doesn't use a screen reader, but a sighted keyboard user who doesn't use a screen reader would never find that information.

**Comment by majapw** [comment]
> However, I do find the content that gets read out, and the general instructions, to be somewhat problematic for accessibility in that the instructions are only valid for a keyboard user who doesn't use a screen reader, but a sighted keyboard user who doesn't use a screen reader would never find that information.

That seems like a different issue that we can work on improving. :) I think as we continue to make improvements, we will get there. 

In any case, this issue seems resolved. I'm gonna close it!
